e2e/utils.go

```diff
@@ -299,6 +299,19 @@ func validateNormalUserPVCAccess(pvcPath string, f *framework.Framework) error {
 	if stdErr != "" {
 		return fmt.Errorf("failed to touch a file as non-root user %v", stdErr)
 	}
+
+	// metrics for BlockMode was added in Kubernetes 1.22
+	isBlockMode := false
+	if pvc.Spec.VolumeMode != nil {
+		isBlockMode = (*pvc.Spec.VolumeMode == v1.PersistentVolumeBlock)
+	}
+	if !isBlockMode || k8sVersionGreaterEquals(f.ClientSet, 1, 22) {
```

Comment:
This should be for blockMode right? it should be `isBlockMode && k8sVersionGreaterEquals(f.ClientSet, 1, 22)` check?

Reply:
Yes, it should be `&&` .

Reply:
No, even for non-BlockMode volumes we should test the metrics. That was not part of the e2e tests until now, this adds that as well.

Only from Kubernetes 1.22 the metrics for BlockMode PVCs are available.

Reply:
okay, this means we are testing the filesystem mode pvc metrics not the block mode yet.

Reply:
> okay, this means we are testing the filesystem mode pvc metrics not the block mode yet.

Yes, correct.